src/schemas/json/bitrise.json

```diff
@@ -0,0 +1,445 @@
+  "definitions": {
+    "AppModel": {
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "envs": {
+          "items": {
+            "patternProperties": {
+              ".*": {
+                "additionalProperties": true
+              }
+            },
+            "type": "object"
+          },
+          "type": "array"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "AptGetDepModel": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "bin_name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BashStepToolkitModel": {
+      "properties": {
+        "entry_file": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BitriseDataModel": {
+      "required": ["format_version", "project_type"],
+      "properties": {
+        "format_version": {
+          "type": "string"
+        },
+        "default_step_lib_source": {
+          "type": "string"
+        },
+        "project_type": {
+          "type": "string"
+        },
+        "title": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "app": {
+          "$ref": "#/definitions/AppModel"
+        },
+        "meta": {
+          "patternProperties": {
+            ".*": {
+              "additionalProperties": true
+            }
+          },
+          "type": "object"
+        },
+        "trigger_map": {
+          "items": {
+            "$ref": "#/definitions/TriggerMapItemModel"
+          },
+          "type": "array"
+        },
+        "pipelines": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/PipelineModel"
+            }
+          },
+          "type": "object"
+        },
+        "stages": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/StageModel"
+            }
+          },
+          "type": "object"
+        },
+        "workflows": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/WorkflowModel"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BrewDepModel": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "bin_name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "CheckOnlyDepModel": {
+      "properties": {
+        "name": {
```

Comment:
Afaik this has fixed value(s)? (xcode?)

Reply:
Where can I find the possible values / the place of enforcement?